src/util/index.js

```diff
@@ -781,6 +781,202 @@ var util = module.exports = {
     return sumw ? position : {x: p0.x, y: p0.y};
   },
 
+  /**
+   * Get the square of the Euclidean 2D distance between two points.
+   *
+   * @param {geo.geoPosition} pt1 The first point.
+   * @param {geo.geoPosition} pt2 The second point.
+   * @returns {number} The distance squared.
+   */
+  distance2dSquared: function (pt1, pt2) {
```

Comment:
should we just have distance2d? It may be worth using turfjs? (https://turfjs.org/)

Reply:
If we return the non-squared versions, there is a lot of needless sqrt function calls, since mostly we are trying to find the smallest value.

Reply:
turfjs internally does everything on a spheroid.  Their distance function, for instance, is much heavier because of this, always making multiple trigonometric function calls.  Since we are usually already in a 2d spatial domain, this is vastly slower (and doesn't extend to pixel-space examples in a rational manner, or ellipsoids that are highly eccentric, etc.).

Reply:
sure, perhaps in the future we could consider it.